pdf/internal/textencoding/glyphs_glyphlist.go

```diff
@@ -2713,73 +2713,73 @@ var glyphlistGlyphToRuneMap = map[GlyphName]rune{ // 6339 entries
 	"Aybarmenian":                         0x0531,  // Ա '\u0531'
 	"B":                                   0x0042,  // B 'B'
 	"Barv":                                0x2ae7,  // ⫧ '\u2ae7'
-	"BbbA":                                0x1d538, // 𝔸 '\U0001d538'
-	"BbbB":                                0x1d539, // 𝔹 '\U0001d539'
+	"BbbA":                                0x1d538, // � '\U0001d538'
```

Comment:
Seems like the unicode value in the comment was previously displaying but now is represented as an unknown code.  Is that expected?
Was 𝔸 is now �  at least as displayed in GitHub